guitarHarmony/note.py

```diff
@@ -119,4 +119,8 @@ def __str__(self):
 
     def __eq__(self, other):
         if isinstance(other, str): other=Note(other)
-        return self.nameWithOctave == other.nameWithOctave
\ No newline at end of file
+        return self.nameWithOctave == other.nameWithOctave
+
+    def __gt__(self, other):
+        if isinstance(other, str): other=Note(other)
+        return self.nameWithOctave >= other.nameWithOctave
```

Comment:
try use: self.getSemiSteps() >= other.getSemiSteps(). Note that different notes may have the same semisteps..